android/src/main/java/com/rncamerakit/CKCameraManager.kt

```diff
@@ -44,81 +51,93 @@ class CKCameraManager : SimpleViewManager<CKCamera>() {
+    override fun setFocusMode(view: CKCamera, mode: String?) {
+        view.setAutoFocus(mode ?: "on")
     }
 
     @ReactProp(name = "zoomMode")
-    fun setZoomMode(view: CKCamera, mode: String?) {
+    override fun setZoomMode(view: CKCamera, mode: String?) {
         view.setZoomMode(mode)
     }
 
     @ReactProp(name = "zoom", defaultDouble = -1.0)
-    fun setZoom(view: CKCamera, factor: Double) {
+    override fun setZoom(view: CKCamera, factor: Double) {
         view.setZoom(if (factor == -1.0) null else factor)
     }
 
     @ReactProp(name = "maxZoom", defaultDouble = 420.0)
-    fun setMaxZoom(view: CKCamera, factor: Double) {
+    override fun setMaxZoom(view: CKCamera, factor: Double) {
         view.setMaxZoom(factor)
     }
 
     @ReactProp(name = "scanBarcode")
-    fun setScanBarcode(view: CKCamera, enabled: Boolean) {
+    override fun setScanBarcode(view: CKCamera, enabled: Boolean) {
         view.setScanBarcode(enabled)
     }
 
     @ReactProp(name = "showFrame")
-    fun setShowFrame(view: CKCamera, enabled: Boolean) {
+    override fun setShowFrame(view: CKCamera, enabled: Boolean) {
         view.setShowFrame(enabled)
     }
 
     @ReactProp(name = "laserColor", defaultInt = Color.RED)
-    fun setLaserColor(view: CKCamera, @ColorInt color: Int) {
-        view.setLaserColor(color)
+    override fun setLaserColor(view: CKCamera, @ColorInt color: Int?) {
+        view.setLaserColor(color ?: Color.RED)
     }
 
     @ReactProp(name = "frameColor", defaultInt = Color.GREEN)
-    fun setFrameColor(view: CKCamera, @ColorInt color: Int) {
-        view.setFrameColor(color)
+    override fun setFrameColor(view: CKCamera, @ColorInt color: Int?) {
+        view.setFrameColor(color ?: Color.GREEN)
     }
 
     @ReactProp(name = "outputPath")
-    fun setOutputPath(view: CKCamera, path: String) {
-        view.setOutputPath(path)
+    override fun setOutputPath(view: CKCamera, path: String?) {
+        view.setOutputPath(path ?: "")
     }
 
     @ReactProp(name = "shutterAnimationDuration")
-    fun setShutterAnimationDuration(view: CKCamera, duration: Int) {
+    override fun setShutterAnimationDuration(view: CKCamera, duration: Int) {
         view.setShutterAnimationDuration(duration)
     }
 
     @ReactProp(name = "shutterPhotoSound")
-    fun setShutterPhotoSound(view: CKCamera, enabled: Boolean) {
+    override fun setShutterPhotoSound(view: CKCamera, enabled: Boolean) {
         view.setShutterPhotoSound(enabled);
     }
+
+    override fun setRatioOverlay(view: CKCamera?, value: String?) = Unit
+
+    override fun setRatioOverlayColor(view: CKCamera?, value: Int?) = Unit
+
+    override fun setResetFocusTimeout(view: CKCamera?, value: Int) = Unit
+
+    override fun setResetFocusWhenMotionDetected(view: CKCamera?, value: Boolean) = Unit
+
+    override fun setResizeMode(view: CKCamera?, value: String?) = Unit
+
+    override fun setScanThrottleDelay(view: CKCamera?, value: Int) = Unit
```

Comment:
I assume those are iOS-only, if so can you add a comment saying that?

Reply:
Done in 79dfe341899ebcf39e412973cd26538504d370d4